addons/l10n_eg_edi_eta/__manifest__.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "Egyptian E-Invoice Integration",
+    'summary': """
+            Egyptian Tax Authority Invoice Integration
+        """,
+    'description': """
+       This module integrate with the ETA Portal to automatically sign and send your invoices to the tax Authority.
+       Special thanks to Plementus <info@plementus.com>, Mario Roshdy <marioroshdyn@gmail.com> and Karim Jaber <kareem@plementus.com> 
```

Comment:
should the description include personal emails? Websites and names are OK for credits, but personal emails are supposedly private, no?